packages/app-desktop/gui/NoteEditor/utils/contextMenu.test.ts

```diff
@@ -0,0 +1,60 @@
+import { textToDataUri, ContextMenuOptions, ContextMenuItemType, svgUriToPng } from './contextMenuUtils';
+
+jest.mock('@joplin/lib/models/Resource');
+
+describe('contextMenu', () => {
+	it('should provide proper copy path', async () => {
+		const testCases: Array<[ContextMenuOptions, string]> = [
+			[
+				{
+					itemType: ContextMenuItemType.Image,
+					resourceId: null,
+					textToCopy: '<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve">test</svg>',
+					mime: 'image/svg+xml',
+					filename: 'Mermaid export.svg',
+					linkToCopy: null,
+					htmlToCopy: null,
+					insertContent: () => { },
+				},
+				'data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB4bWw6c3BhY2U9InByZXNlcnZlIj50ZXN0PC9zdmc+',
+			],
+		];
+
```

Comment:
You don't need all this. Your function just take some text and a mime type, so your test cases should just be:

`["some text", "mime/type"]`


And you don't need a loop if there's just one test case. Usually multiple test cases are preferred but in this case one is fine.